src/portable/portabled.c

```diff
@@ -79,9 +86,21 @@ static int manager_connect_bus(Manager *m) {
         assert(m);
         assert(!m->bus);
 
-        r = sd_bus_default_system(&m->bus);
+        if (m->runtime_scope == RUNTIME_SCOPE_SYSTEM) {
+                r = sd_bus_default_system(&m->bus);
+                if (r < 0)
+                        return log_error_errno(r, "Failed to connect to system bus: %m");
+        } else {
+                assert(m->runtime_scope == RUNTIME_SCOPE_USER);
+
+                r = sd_bus_default_user(&m->bus);
+                if (r < 0)
+                        return log_error_errno(r, "Failed to connect to user bus: %m");
+        }
```

Comment:
one of these days we should add a generic implementation of this if/else expression